cmp/dimensionchooser/DimensionChooserModel.js

```diff
@@ -82,6 +88,7 @@ export class DimensionChooserModel {
         this.maxDepth = maxDepth;
         this.enableClear = enableClear;
         this.preference = preference;
+        this.historyPreference = withDefault(historyPreference, preference);
```

Comment:
This bit of defaulting can be done right in the constructor -- you can refer to previous arguments when defining subsequent ones -- very cool feature of javascript